tools/extras/install_irstlm.sh

```diff
@@ -55,6 +55,7 @@ fi
   [ ! -z ${IRSTLM} ] && \
     echo >&2 "IRSTLM config is already in env.sh" && exit
 
+  wd=`pwd`
   wd=`readlink -f $wd 2>/dev/null || pwd`
```

Comment:
Reading http://www.unix.com/man-page/posix/1posix/pwd/ probably the desired behaviour is just one line:
```
wd=`pwd -P`
```

or, for maximum compatibility with already used systems:

```
wd=`pwd -P 2>/dev/null || pwd`
```

This will avoid the unportable ```readlink -f```, provide a portable solution and will be equivalent to or better than (from the performance point of view, that is completely irrelevant in this case) ```readlink -f $PWD``` in bash where ```pwd``` is a builtin.